test/flows/business_coronavirus_support_finder_flow_test.rb

```diff
@@ -0,0 +1,255 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class BusinessCoronavirusSupportFinderFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup { testing_flow BusinessCoronavirusSupportFinderFlow }
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: business_based?" do
+    setup { testing_node :business_based? }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[england scotland wales northern_ireland].each do |response|
```

Comment:
As per the review on FindCoronavirusSupport just testing one of these should suffice since there is no logic in the Flow for this.

Reply:
Thanks, I should have been clearer but this comment applies to all of the places you've got a loop of responses - as they're all in the same situation where they don't have any logic for the next node.